packages/flutter_tools/gradle/aar_init_script.gradle

```
@@ -175,8 +175,13 @@ projectsEvaluated {
             // Configure the `assembleAar<variantName>` task for each plugin's projects and make
             // the module's equivalent task depend on the plugin's task.
             String variantName = variant.name.capitalize()
-            moduleProject.tasks.findByPath("assembleAar$variantName")
-                .dependsOn(pluginProject.tasks.findByPath("assembleAar$variantName"))
+            try {
+                Task mpt = moduleProject.tasks.named("assembleAar$variantName").get()
```

`mpt` : module project task
`ppt` : plugin project task

---

I think this would be better as the full names
```suggestion
                Task moduleProjectTask = moduleProject.tasks.named("assembleAar$variantName").get()
                Task pluginProjectTask = pluginProject.tasks.named("assembleAar$variantName").get()
```

https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-abbreviations